db/migrate/20190214113305_add_index_views_using_warehouse_item_id.rb

```diff
@@ -0,0 +1,36 @@
+    remove_index :aggregations_search_last_months, name: :search_last_month_gin_base_path_upviews
+    remove_index :aggregations_search_last_months, name: :search_last_month_gin_base_path_organisation_id
+    remove_index :aggregations_search_last_months, name: :search_last_month_gin_base_path_document_type
+    remove_index :aggregations_search_last_thirty_days, name: :search_last_thirty_days_gin_base_path_upviews
+    remove_index :aggregations_search_last_thirty_days, name: :search_last_thirty_days_gin_base_path_organisation_id
+    remove_index :aggregations_search_last_thirty_days, name: :search_last_thirty_days_gin_base_path_document_type
+
+
+    add_index :aggregations_search_last_twelve_months, [:upviews, :warehouse_item_id], name: :search_last_twelve_months_gin_base_path_upviews
+    add_index :aggregations_search_last_twelve_months, [:organisation_id, :upviews, :warehouse_item_id], name: :search_last_twelve_months_gin_base_path_organisation_id
+    add_index :aggregations_search_last_twelve_months, [:document_type, :upviews, :warehouse_item_id], name: :search_last_twelve_months_gin_base_path_document_type
+    add_index :aggregations_search_last_six_months, [:upviews, :warehouse_item_id], name: :search_last_six_months_gin_base_path_upviews
+    add_index :aggregations_search_last_six_months, [:organisation_id, :upviews, :warehouse_item_id], name: :search_last_six_months_gin_base_path_organisation_id
+    add_index :aggregations_search_last_six_months, [:document_type, :upviews, :warehouse_item_id], name: :search_last_six_months_gin_base_path_document_type
+    add_index :aggregations_search_last_three_months, [:upviews, :warehouse_item_id], name: :search_last_three_months_gin_base_path_upviews
+    add_index :aggregations_search_last_three_months, [:organisation_id, :upviews, :warehouse_item_id], name: :search_last_three_months_gin_base_path_organisation_id
+    add_index :aggregations_search_last_three_months, [:document_type, :upviews, :warehouse_item_id], name: :search_last_three_months_gin_base_path_document_type
+    add_index :aggregations_search_last_months, [:upviews, :warehouse_item_id], name: :search_last_month_gin_base_path_upviews
+    add_index :aggregations_search_last_months, [:organisation_id, :upviews, :warehouse_item_id], name: :search_last_month_gin_base_path_organisation_id
+    add_index :aggregations_search_last_months, [:document_type, :upviews, :warehouse_item_id], name: :search_last_month_gin_base_path_document_type
+    add_index :aggregations_search_last_thirty_days, [:upviews, :warehouse_item_id], name: :search_last_thirty_days_gin_base_path_upviews
+    add_index :aggregations_search_last_thirty_days, [:organisation_id, :upviews, :warehouse_item_id], name: :search_last_thirty_days_gin_base_path_organisation_id
+    add_index :aggregations_search_last_thirty_days, [:document_type, :upviews, :warehouse_item_id], name: :search_last_thirty_days_gin_base_path_document_type
```

Comment:
I don't understand the naming behind these indexes? i.e. `search_last_twelve_months_gin_base_path_upviews` what does `gin_base_path` stand for?

Reply:
Good spot. I have removed it as they were copy pasted.